packages/p2p/src/components/floating-rate/floating-rate.jsx

```diff
@@ -15,22 +15,34 @@ const FloatingRate = ({
     error_messages,
     fiat_currency,
     local_currency,
+    onChangeHandler,
```

Comment:
Same, better call it `onChange` or `handleChange`

Reply:
Renamed the function